src/gui/static/src/app/templates/wallet.html

```diff
@@ -596,20 +596,26 @@ <h4 class="modal-title">Create New Wallet</h4>
                 <button type="button" (click)="hideLoadSeedWalletPopup()" class="close">&times;</button>
                 <h4 class="modal-title">Load wallet by seed</h4>
             </div>
-            <form class="form-horizontal" (ngSubmit)="createWalletSeed(walletname.value, walletseed.value)">
+            <form class="form-horizontal" (ngSubmit)="createWalletSeed(walletname.value, walletseed.value, addressesToCreate.value)">
                 <div class="modal-body">
                     <div class="form-group">
-                        <label class="col-sm-2 control-label">Name:</label>
+                        <label class="col-sm-3 control-label">Name:</label>
```

Comment:
`New Wallet` use `Label` field, we should have consistent field name, I propose change the `Label` field in `New Wallet` to `Name`.